doc/BuildProgram.md

```diff
@@ -0,0 +1,64 @@
+# FreeRCT build instructions #
+
+Currently only Linux is officially supported, but Windows support should be possible with a few minor modifications. See [BuildProgramVisualStudio](BuildProgramVisualStudio.md) for instructions on building FreeRCT on Windows using Visual Studio.
+
+## Checkout ##
+
+The source code is hosted on GitHub, so you will need a git client to get it.
+
+A command to do this is `git clone https://github.com/FreeRCT/FreeRCT`.
+
+## Building the program ##
+
+Almost everything is written in C++, which means you need **g++** or **clang++** to compile it. FreeRCT uses C++11 features, so g++ 4.8+ or clang 3.3+ is recommended.
+In addition, you need:
+
+  * **libpng** - Making the RCD data files that contain the graphics and other data read by the program.
+  * **SDL2** & **SDL2\_ttf** - Displaying graphics of the program. Note that SDL2 versions of the libraries are required.
+  * **CMake>=2.8** & **make** - Building the program.
+  * `[optional]` **lex/flex** - Scanner generator for generating RCD input files.
+  * `[optional]` **yacc/bison** - Parser generator for generating RCD input files.
+
+The existence of these programs/libraries is checked by `cmake`.
+
+Building is as simple as
+
+```
+$ cd FreeRCT # Go into the downloaded source directory
```

Comment:
Maybe reference to some unique file that's here (a cmake one would be pretty stable I think)